src/ServiceStack.OrmLite.Oracle.Tests/OracleParamTests.cs

```diff
@@ -269,328 +259,922 @@ public void ORA_ParamByteTest()
         }
 
         [Test]
-        public void ORA_ReservedNameComment_Test()
+        public void ORA_ReservedNameLevelAsAlias_Test()
         {
             using (var db = OpenDbConnection())
             {
                 DropAndCreateTables(db);
 
-                var row = new ParamComment { Comment = 1, Id = 2 };
+                var row = new ParamLevelAsAlias { Id = 2, ApprovalLevel = 212218 };
                 db.Insert(row);
 
-                row.Comment = 454;
+                row.ApprovalLevel = 676;
                 db.Update(row);
             }
         }
 
+        [ImportMany(typeof(IParam))]
```

Comment:
It looks like you're adding the first usage of System.ComponentModel.Composition in the codebase. I'm not sure how @mythz will feel about that.